### PR TITLE
POWR-2224 Message styling updates

### PR DESCRIPTION
**Pull request by brittanysmart**
- Update Legacy Path and Back Reference blocks to use default message style
- Update warning message icon
- Clean up message styling
- Add consistent border-radius across messages

### CODE REVIEW

File: web/themes/custom/cloudy/src/components/message/message.scss [hunk 1]
```diff
@@ -2,42 +2,28 @@
...
 }
 
 .message {
   display: block;
-  margin-bottom: 2px;
   padding: $cloudy-space-4;
-  color: $cloudy-color-neutral-0;
+  color: $cloudy-color-neutral-900;
+  border-radius: $cloudy-space-0;
+  border: none !important; // BAD, but only way to override Bootstrap
```
Comment (joshuami) [inline web/themes/custom/cloudy/src/components/message/message.scss:18]: Do you need the !important if you aren't using a Boostrap class anymore?

File: web/modules/custom/portland/templates/block--portland-revision-block.html.twig [hunk 2]
```diff
@@ -5,10 +5,10 @@
  */
 #}
 {% if current_revision_state %}
-  <div class="portland-revision--block message message-{{ alert_color }} border border-{{ alert_color }} mt-3">
-    {# <h1>{{ title }}</h1> #}
-    <span class="fas fa-lg fa-{{ alert_icon }} fa-fw"></span>
-    <strong>{{ current_revision_state }}</strong> {{ latest_revision_state }}
+  <div class="portland-revision--block message message-{{ alert_color }} mt-3">
+    <span class="fas fa-md fa-{{ alert_icon }} fa-fw"></span>
```
Comment (joshuami) [inline web/modules/custom/portland/templates/block--portland-revision-block.html.twig:9]: I was surprised to see that this didn't get an SVG for the icons.

Comment (joshuami) [reply web/modules/custom/portland/templates/block--portland-revision-block.html.twig:9]: And I'm also okay with that being a follow up.

File: web/modules/custom/portland/modules/portland_edit_forms/back_reference_block.inc [hunk 3]
```diff
@@ -27,11 +27,11 @@ function portland_edit_forms_form_node_form_alter(&$form, $form_state, $form_id)
 
     $form['content_back_reference'] = [
       '#type' => 'container',
-      '#attributes' => ['class' => ['alert', 'alert-info', 'border', 'border-info']],
+      '#attributes' => ['class' => ['message']],
     ];
     $form['content_back_reference']['title'] = [
       '#type' => 'inline_template',
-      '#template' => '<label>{{ title|t }}</label>',
```
Comment (joshuami) [inline web/modules/custom/portland/modules/portland_edit_forms/back_reference_block.inc:34]: I'm glad you caught this!

File: web/modules/custom/portland/src/Plugin/Block/RevisionBlock.php [hunk 4]
```diff
@@ -21,8 +21,8 @@
 class RevisionBlock extends BlockBase {
     private static $severity_icon = [
       'danger' => 'ban',
-      'warning' => 'exclamation-circle',
-      'success' => 'info-circle'
+      'warning' => 'exclamation-triangle',
```
Comment (joshuami) [inline web/modules/custom/portland/src/Plugin/Block/RevisionBlock.php:24]: I'm actually okay with exclamation-circle in this case. The triangle is more of a warning where the circle is more of a notice? I would totally approve this either way, but wanted to get this into place.

Comment (brittanysmart) [reply web/modules/custom/portland/src/Plugin/Block/RevisionBlock.php:24]: I hear what you're saying. I'm going to create a message component story so we can then swap out message instances for a component. I'll make a note to revisit the icon choice/intention, warning vs. notice usage.

File: web/themes/custom/cloudy/src/components/message/message.scss (hunk 1 above)
Comment (brittanysmart) [reply web/themes/custom/cloudy/src/components/message/message.scss]: Nope. It's gone :)

**Pull request merged**